android/src/main/java/com/swmansion/rnscreens/ScreenContainer.java

```diff
@@ -328,13 +338,13 @@ private final void onUpdate() {
     mFragmentManager.executePendingTransactions();
 
     performUpdate();
+    notifyContainerUpdate();
```

Comment:
shouldn't this be called directly from `perfomUpdate`? Are there any other ways `performUpdate` method is called when we don't want `notify` to be triggered? If not why?

Reply:
I think not. I added this here since we call it both in `ScreenContainer` and `ScreenStack` this way and if it is in `perfomUpdate`, then we need to use this method in both classes. I don't have a strong opinion on this.